roles/jenkins/tasks/azure-agents.yml

```diff
@@ -1,10 +1,10 @@
 ---
 
-- name: Install Jenkins credential using script for the {{ credential.id }} secret.
+- name: Configure Azure Agent VM - generic profile used with legacy SPs login.
```

Comment:
is this change needed?

Reply:
yes

Reply:
this file should be removed with changing over to azure vm agent